cfme/tests/automate/test_domain.py

```diff
@@ -453,3 +453,65 @@ def test_tenant_attached_with_domain(request, new_user, domain):
                 assert domain['Tenant'] == tenant.name
             else:
                 assert domain['Tenant'] == "My Company"
+
+
+@pytest.fixture(scope='module')
+def user(appliance):
```

Comment:
Q: Do you think it is worth adding these (`role`, `user`, `group`) as global fixtures to `cfme.fixtures`? Will they be used elsewhere?

Reply:
Okay, thanks for the detailed explanation @ganeshhubale, since the roles and permissions are specific to the test, I agree that these should not be global fixtures. Especially since there is already a global fixture for these that creates them via REST.